.eslintrc.json

```
@@ -112,16 +112,14 @@
         "no-useless-constructor": "off",
         "@typescript-eslint/no-useless-constructor": "warn",
         "@typescript-eslint/prefer-literal-enum-member": "error"
-      },
-      "overrides": [
-        {
-          "files": ["packages/**"],
-          "rules": {
-            "jsdoc/no-types": "error",
-            "jsdoc/no-undefined-types": "error"
-          }
-        }
-      ]
+      }
+    },
+    {
+      "files": ["packages/**"],
```

```suggestion
      "files": ["packages/**/*.tsx?"],
```

According to their docs:

> Prohibits types on @param or @returns (redundant with TypeScript)

So we should make sure this is only run against TypeScript files, using it in JavaScript should still be fine.